enhancements/api-review/add-new-CRD-ImageDigestMirrorSet-and-ImageTagMirrorSet-to-config.openshift.io.md

```diff
@@ -326,6 +326,30 @@ Phase 2:
 - components listed in the above [document](https://docs.google.com/document/d/11FJPpIYAQLj5EcYiJtbi_bNkAcJa2hCLV63WvoDsrcQ/edit?usp=sharing) migrate to new APIs ImageDigestMirrorSet, ImageTagMirrorSet
 - register and expose the CRDs.
 
+#### Implementation updates
+Actual implementation for migration path of Phase 2 was different from the original design:
+
+The webhook is not implementable. It cannot convert between different kind of resources. 
+
+If copy existing ImageContentSourcePolicy content to ImageDigestMirrorSet allow both kind of objecting existing, this meeting [notes](https://docs.google.com/document/d/13h6IJn8wlzXdiPMvCWlMEHOXXqEZ9_GYOl02Wldb3z8/edit?usp=sharing) raised the concern that in some cases the final `registries.conf` will lead the customer implying only one of the object is honored.
+
+For the above reason the actual implementation for migration path:
+- implement the migration path:
+  - [openshift/kubernetes#1310](https://github.com/openshift/kubernetes/pull/1310) prohibit coexistence of ICSP and IDMS objects, or ICSP and ITMS objects.
+  - [openshift/oc#1238](https://github.com/openshift/oc/pull/1238) implemented oc command that can convert ImageContentSourcePolicy yaml to ImageDigestMirrorSet yaml. The command can be used by customer when they want to migrate from ICSP to IDMS resources.
+
+#### Update the implementation regarding the customer requests
```

Comment:
This section should say that ICSP is a Tier 1 API and so we support it during all of 4.x. Link to https://docs.openshift.com/container-platform/4.13/rest_api/understanding-api-support-tiers.html.
This is the motivation for continuing to support ICSP but we still mark it as deprecated and encourage users to migrate to IDMS.